### PR TITLE
RS+RT/YJ/RULE 22-27

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**
I'd appreciate it if you reviewed this PR.

### CODE REVIEW

**Review by charliepnnl — APPROVED** [review]
Approved with questions about naming.

File: rct229/rules/section22/section22rule27.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        baseline_system_types = {
+            "SYS-7": ["hvac_sys_7"],
+            "SYS-11.1": ["hvac_sys_11_1"],
+        }
+        # if any system type found in the APPLICABLE_SYS_TYPES then return applicable.
+        return any(
+            [key in APPLICABLE_SYS_TYPES for key in baseline_system_types.keys()]
+        )
+
+    class ChillerFluidLoopRule(RuleDefinitionBase):
```
[the remark this answers is not in the document]
Comment (yunjoonjung-PNNL) [reply rct229/rules/section22/section22rule27.py:47]: Fixed.

File: rct229/rules/section22/section22rule27.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        def __init__(self):
+            super(Section22Rule27.ChillerFluidLoopRule, self).__init__(
+                rmrs_used=UserBaselineProposedVals(False, True, False),
+                required_fields={
+                    "$": ["is_condenser_water_pump_interlocked"],
+                },
+            )
+
+        def get_calc_vals(self, context, data=None):
+            fluid_loop_b = context.baseline
```
[the remark this answers is not in the document]
Comment (yunjoonjung-PNNL) [reply rct229/rules/section22/section22rule27.py:57]: Fixed

File: rct229/rules/section22/section22rule27.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+
+APPLICABLE_SYS_TYPES = [
```
Comment (weilixu) [inline rct229/rules/section22/section22rule27.py:5]: The `get_baseline_system_types` is merged. Please update the `APPLICABLE_SYS_TYPES` list and the `is_applicable` function.

Comment (yunjoonjung-PNNL) [reply rct229/rules/section22/section22rule27.py:7]: Fixed.

**Review by weilixu — APPROVED** [review]

**Pull request merged**